ci-operator/config/openshift-kni/telco-reference/openshift-kni-telco-reference-release-4.16.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM registry.ci.openshift.org/ocp/ubi-python-311:9
```

Comment:
dont you prefer to have the dockerfile in the original telco-reference repo?

Reply:
I think I would prefer this, but @imiller0 didn't set it up that way.

Reply:
I put the dockerfile here/inline and enabled use_build_cache below in an attempt to speed up the CI. It would be interesting to compare the average CI time using this model vs dockerfile in the repo.

Reply:
I when I said I would prefer the in-repo dockerfile, I wasn't thinking about it from a "speed of CI run" but more of a "maintainability" perspective.  If there is a speed-up I'd love to know...

Reply:
the maintainability aspect, is that it supposed to be easier for the project developers to maintain that file
I suggest also using it in the Makefile build 
so it is being used and exercised regularly